nuitka/plugins/standard/ImplicitImports.py

```diff
@@ -722,4 +730,4 @@ def decideCompilation(self, module_name, source_ref):
             if module_name == unworthy_namespace or module_name.startswith(
                 unworthy_namespace + "."
             ):
-                return "bytecode"
```

Comment:
Stray new line change at end of file (that the commit hook is not yet dealing with), please remove it while you change the file again.

Reply:
Hi, looks like this was automatically done by PyCharm, I'll take a look.